firmware/main.c

```diff
@@ -1,18 +1,36 @@
-/* 
- * Project: Micronucleus -  v2.04
+/*
+ * Project: Micronucleus -  v2.4
  *
- * Micronucleus V2.04             (c) 2016 Tim Bo"scke - cpldcpu@gmail.com
+ * Micronucleus V2.4             (c) 2020 Armin Joachimsmeyer armin.joachimsmeyer@gmail.com
+ * Micronucleus V2.04            (c) 2016 Tim Bo"scke - cpldcpu@gmail.com
+ * Micronucleus V2.3             (c) 2016 Tim Bo"scke - cpldcpu@gmail.com
  *                               (c) 2014 Shay Green
  * Original Micronucleus         (c) 2012 Jenna Fox
  *
  * Based on USBaspLoader-tiny85  (c) 2012 Louis Beaudoin
  * Based on USBaspLoader         (c) 2007 by OBJECTIVE DEVELOPMENT Software GmbH
  *
- * License: GNU GPL v2 (see License.txt)
+ *  This file is part of micronucleus https://github.com/micronucleus/micronucleus.
+ *
+ *  Micronucleus is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/gpl.html>.
  */
- 
+
 #define MICRONUCLEUS_VERSION_MAJOR 2
-#define MICRONUCLEUS_VERSION_MINOR 4
+#define MICRONUCLEUS_VERSION_MINOR 4 // 164 (0xA4) is shown in W10 Device manager in BCD but as :4 instead of A4
```

Comment:
I don't understand this comment. Should something be changed?

Reply:
Long ago, let me think....

Reply:
I can't remember what the tool or attribute  was where I saw this data. So the comment should be removed at the next commit!